local-notifications/src/web.ts

```diff
@@ -94,6 +94,26 @@ export class LocalNotificationsWeb
     return { display };
   }
 
+  protected checkNotificationSupport = (): boolean => {
+    if (!window.Notification || !Notification.requestPermission) {
```

Comment:
```suggestion
    if (!('Notification' in window) || !Notification.requestPermission) {
```
Why change this? I don't prefer one over the other, we should be consistent with this window checks, most plugins use it like this

Reply:
Just for consistency with the requestPermission check.  It doesn't really make a difference - I can change it back.